lib/pghero.rb

```diff
@@ -19,13 +24,14 @@
 require "pghero/methods/tables"
 require "pghero/methods/users"
 
+require "pghero/base_database"
 require "pghero/database"
 require "pghero/engine" if defined?(Rails)
+require "pghero/pg_const"
+require "pghero/repository"
 require "pghero/version"
 
 module PgHero
-  autoload :Connection, "pghero/connection"
-  autoload :QueryStats, "pghero/query_stats"
```

Comment:
Is there a reason to disable autoloading, without requiring these? Are these effectively dead code now?